docs/lib/restapi/fragments/flutter/getting-started.md

```diff
@@ -0,0 +1,50 @@
+The Amplify CLI deploys REST APIs and handlers using [Amazon API Gateway](http://docs.aws.amazon.com/apigateway/latest/developerguide/) and [AWS Lambda](http://docs.aws.amazon.com/lambda/latest/dg/).
+
+See the authentication section for more details for using the `AWSMobileClient` in your application.
+
+## Create new REST API
+
+In a terminal window, navigate to your project folder (the folder that contains your app `/lib` folder), and add the SDK to your app.
```

Comment:
SDK should be changed to 'library`

Reply:
getting-started.md shouldn't be included.